tools/build_and_deploy_to_folgers.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Intended to be executed on a builder machine with ssh forwarding to folgers; e.g., Atrium.